daemon/daemon.go

```diff
@@ -880,9 +877,18 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine) (*Daemon, error)
 	}
 	log.Debugf("Using graph driver %s", driver)
 
-	// As Docker on btrfs and SELinux are incompatible at present, error on both being enabled
-	if selinuxEnabled() && config.EnableSelinuxSupport && driver.String() == "btrfs" {
-		return nil, fmt.Errorf("SELinux is not supported with the BTRFS graph driver!")
+	if config.EnableSelinuxSupport {
+		if selinuxEnabled() {
+			// As Docker on btrfs and SELinux are incompatible at present, error on both being enabled
+			if driver.String() == "btrfs" {
+				return nil, fmt.Errorf("SELinux is not supported with the BTRFS graph driver")
+			}
+			log.Debugf("SELinux enabled successfully")
```

Comment:
Ok,i see,i have checked the functions of Debug and Debugf, then understanded difference.
In addition, i have checked the whole daemon/daemon.go, and found the same situation of mine, just like "log.Debugf("Creating images graph")" and so on, so maybe we can use "log.Debug" to replace "log.Debugf", what's your opinion?Thanks:-)